package.json

```diff
@@ -62,6 +62,7 @@
     }
   ],
   "dependencies": {
+    "babel-polyfill": "^6.23.0",
```

Comment:
and i aggree this don't needs a pollyfill i think as we don't load it :) and build directly